src/docs/slot_specifications.md

```diff
@@ -0,0 +1,299 @@
+
+The data or information a term (slot) encodes MUST be in the form of a valid LinkML `range:` type:
+
+- `string`
+- `integer`
+- `float`
+- `boolean`
+- A MIxS defined enumeration
+
+Refer to LinkML documentation for more information on [range types](https://linkml.io/linkml-model/latest/docs/range/).
+
+## 3 Slot attributes
+
+### 3.1. Minimal required LinkML slot attributes
+
+A term (slot) MUST at a minimum include following attributes:
+
+- [`description`](https://linkml.io/linkml/schemas/metadata.html#providing-descriptions).
+- [`title`](https://linkml.io/linkml-model/latest/docs/title/).
+- [`examples`](https://linkml.io/linkml-model/latest/docs/examples/).
+- [`in_subset`](https://linkml.io/linkml-model/latest/docs/in_subset/).
+- [`keywords`](https://linkml.io/linkml-model/latest/docs/keywords/).
+- [`slot_uri`](https://linkml.io/linkml-model/latest/docs/slot_uri/).
+- [`range`](https://linkml.io/linkml/schemas/slots.html#ranges).
+
+### 3.2. Recommended LinkML slot attributes
+
+A term (slot) that has some level of 'requirement' (mandatory, conditional mandatory, optional) SHOULD include the following LinkML attributes:
+
+- [`recommended`](https://linkml.io/linkml/schemas/slots.html#recommended)
+- [`required`](https://linkml.io/linkml/schemas/slots.html#required)
+
+## 4. Term description
+
+### 4.1 Description contents
+
+The definition (description) SHOULD aim to be precise enough for a user to understand the data the term (slot) is intended to hold, how it should be filled, and used.
+
+Links to external resources (e.g. ontologies, databases, or other documentation) SHOULD be included in the definition (description) when relevant.
+
+### 4.2 Description length
+
+The definition (description) MUST be at a minimum 1 sentence long that is longer than the term (slot) title.
+
+The definition (description) MAY be multiple sentences long, but should be as concise as possible to ensure readability.
+
+### 4.3 Description examples
+
+The definition (description) SHOULD NOT include basic examples of the data the term (slot) is intended to hold (this is covered by the `examples` attribute).
+
+The definition (description) MAY include examples when the information for the term (slot) requires different formatting depending on certain conditions. The definition (description) MAY also include examples when it requires additional understanding that cannot be inferred by looking purely at the `examples` section.
+
+### 4.4 Description external resources
+
+Links or URLs used in the definition (description) to point a reader to an external resource MUST be valid and generally accessible via the public world wide web.
+
+External resources SHOULD only be referred to when from a stable and established resource (i.e., not a personal or website, or a resource that is not widely used).
+
+URLs in external resources specified within descriptions SHOULD also be defined within a LinkML `see_also` slot attribute.
+
```

Comment:
I think its probably worth adding the 'see_also' row to the terminology table at the top (even though there is no MIxS equivelent)

Reply:
The terminology table is only for mapping MIxS to LinkML terms not desribing every jargon - but I've added a direct link to the LinkML docs on that attribute instead :)